### PR TITLE
DOCS-1519: Make `GetImage()` changes in the Python SDK 

### PR DESCRIPTION
**Pull request by sguequierre**
* Update return type and information about decoding
* Change in tutorials where applicable 
* Add to changelog

### CODE REVIEW

File: docs/tutorials/services/webcam-line-follower-robot.md [hunk 1]
```diff
@@ -451,12 +451,13 @@ The code you are using has several functions:
       center of the robot.
       """
```
Comment (sguequierre) [inline docs/tutorials/services/webcam-line-follower-robot.md:452]: This tutorial doesn't have a place with imports

**Review by npentrel — APPROVED** [review]
This looks good to me - thank you for taking this on. Please do get an sme review as well. The release has been pushed to after the holidays so we have a bit of time

**Review by purplenicole730 — APPROVED** [review]
Some notes about the release of this breaking change: The camera breaking change isn't going to be coming out this coming week. Right now, there are more talks going on about adding everything all at once. This means that some of these code changes will not be necessary, i.e. creating a `RawImage` object with the return of `get_image()` calls.
Code-wise, everything looks good to me.

**Comment by sguequierre** [comment]
@purplenicole730 Is there a set time now that these changes are scheduled to be released? Want to make sure this merges then!

**Comment by purplenicole730** [comment]
> @purplenicole730 Is there a set time now that these changes are scheduled to be released? Want to make sure this merges then!

The LGTM was approved this week. There is no set time as of yet as there is still a lot of work to go.

File: docs/tutorials/services/webcam-line-follower-robot.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ To build your own line follower robot, you need the following hardware:
 <!-- prettier-ignore -->
 | Hardware | Avg. price |
 | -------- | ----------------- |
-| **A single board computer**: This tutorial uses a Raspberry Pi running a 64-bit Linux distribution. If you use a different board, configure your {{< glossary_tooltip term_id="board" text="board" >}} using the [instructions for your board](/get-started/installation/#prepare-your-board). | $60 |
+| **A single-board computer**: This tutorial uses a Raspberry Pi running a 64-bit Linux distribution. If you use a different board, configure your {{< glossary_tooltip term_id="board" text="board" >}} using the [instructions for your board](/get-started/installation/#prepare-your-board). | $60 |
```
Comment (sguequierre) [inline docs/tutorials/services/webcam-line-follower-robot.md:54]: I have no idea why this change was introduced-- would keep though?

File: docs/tutorials/projects/send-security-photo.md [hunk 3]
```diff
@@ -215,6 +215,7 @@ import os
 import yagmail
 
 from viam.robot.client import RobotClient
+from viam.components.camera import Camera
```
Comment (sguequierre) [inline docs/tutorials/projects/send-security-photo.md:218]: this was missing

File: docs/appendix/changelog.md [hunk 4]
```diff
@@ -19,6 +19,13 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog date="2024-05-09" color="changed" title="Return type of GetImage()" %}}
+
+The Python SDK introduced a new image container class called [`ViamImage`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage).
+The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [GetDetections()](/ml/vision#getdetections) and [GetClassifications()](/ml/vision#getclassifications) take in `ViamImage` as a parameter.
```
Comment (sguequierre) [inline docs/appendix/changelog.md:25]:
```suggestion
The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [GetDetections()](/ml/vision/#getdetections) and [GetClassifications()](/ml/vision/#getclassifications) take in `ViamImage` as a parameter.
```

File: docs/components/camera/_index.md [hunk 5]
```diff
@@ -136,7 +132,7 @@ frame = await my_camera.get_image(mime_type="image/jpeg")
 
 <br>
 
-If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`RawImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
+If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`ViamImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
```
Comment (purplenicole730) [inline docs/components/camera/_index.md:135]: If it matters, `ViamImage` is still defined in `viam/media/video`. It is also imported by `camera` as well, so also not completely wrong? Not sure if there's a preference!

**Review by purplenicole730 — APPROVED** [review]
LGTM!

File: docs/appendix/changelog.md [hunk 6]
```diff
@@ -19,6 +19,13 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog date="2024-05-09" color="changed" title="Return type of GetImage()" %}}
+
+The Python SDK introduced a new image container class called [`ViamImage`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage).
+The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [GetDetections()](/ml/vision/#getdetections) and [GetClassifications()](/ml/vision/#getclassifications) take in `ViamImage` as a parameter.
+
```
Comment (npentrel) [inline docs/appendix/changelog.md:26]: do we need to provide a code sample for how to use this now? I think there's a helper method to convert a viamimage to a pilimage, we should add that here otherwise people won't know - info on this is on DOCS-2218

Comment (sguequierre) [reply docs/appendix/changelog.md:26]: Added code sample!

File: docs/tutorials/services/color-detection-scuttle.md [hunk 7]
```diff
@@ -317,16 +322,19 @@ You could also write some code with a Viam SDK to [make your rover move in a squ
 
 {{< snippet "social.md" >}}
 
-## Full Code
+## Full code
 
 ```python {class="line-numbers linkable-line-numbers"}
 import asyncio
+import random
+import vlc
```
Comment (sguequierre) [inline docs/tutorials/services/color-detection-scuttle.md:330]: I noticed this was missing and we directed people to import them, but I'm a bit confused as to where they're actually being used?  @npentrel

File: docs/tutorials/projects/send-security-photo.md [hunk 8]
```diff
@@ -251,8 +252,10 @@ async def main():
                 found = True
         if found:
             print("sending a message")
+            # Convert to PIL image
+            pil_image = viam_to_pil_image(img)
```
Comment (npentrel) [inline docs/tutorials/projects/send-security-photo.md:256]: you forgot the import for this I think

File: docs/tutorials/services/color-detection-scuttle.md [hunk 9]
```diff
@@ -176,17 +176,20 @@ python3 main.py
 
 The program prints a list of robot resources.
 
-On top of the packages that the code sample snippet imports, add the `random` and the `vlc` package to the imports.
+On top of the packages that the code sample snippet imports, add the `random` and the `vlc` packages to the imports, and `pil_to_viam_image` and `viam_to_pil_image` from `viam.media.utils.pil`
```
Comment (npentrel) [inline docs/tutorials/services/color-detection-scuttle.md:179]: Yeah I think this got erroneously copied from the guardian tutorial...
```suggestion
On top of the packages that the code sample snippet imports, import `pil_to_viam_image` and `viam_to_pil_image` from `viam.media.utils.pil`.
```

File: docs/tutorials/services/color-detection-scuttle.md [hunk 10]
```diff
@@ -176,17 +176,20 @@ python3 main.py
...
 The top of your code should now look like this:
 
 ```python {class="line-numbers linkable-line-numbers"}
 import asyncio
+import random
+import vlc
```
Comment (npentrel) [inline docs/tutorials/services/color-detection-scuttle.md:185]:
```suggestion
```

File: docs/tutorials/services/color-detection-scuttle.md [hunk 11]
```diff
@@ -317,16 +323,19 @@ You could also write some code with a Viam SDK to [make your rover move in a squ
 
 {{< snippet "social.md" >}}
 
-## Full Code
+## Full code
 
 ```python {class="line-numbers linkable-line-numbers"}
 import asyncio
+import random
+import vlc
```
Comment (npentrel) [inline docs/tutorials/services/color-detection-scuttle.md:331]:
```suggestion
```

**Review by npentrel — COMMENTED** [review]
Thank you, some small changes then good to go

**Review by npentrel — APPROVED** [review]

**Comment by sguequierre** [comment]
> I think this is ready to merge now right?

looks like it!
